src/setuptools_scm/config.py

```diff
@@ -8,7 +8,7 @@
 
 DEFAULT_TAG_REGEX = r"^(?:[\w-]+-)?(?P<version>[vV]?\d+(?:\.\d+){0,2}[^\+]+)(?:\+.*)?$"
 DEFAULT_VERSION_SCHEME = "version_scheme"
-
+DEFAULT_MATCH = '*.*'
```

Comment:
unless there is a scm-portable version of this, we should use a single declaration for the complete git describe command